pkgs/development/python-modules/thefuzz/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, python-Levenshtein, pycodestyle, hypothesis, pytest }:
+
+buildPythonPackage rec {
+  pname = "thefuzz";
+  version = "0.19.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-b3Em2y8silQhKwXjp0DkX0KRxJfXXSB1Fyj2Nbt0qj0=";
+  };
+
+  propagatedBuildInputs = [ python-Levenshtein ];
+  checkInputs = [ pycodestyle hypothesis pytest ];
+
+  meta = with lib; {
+    description = "Fuzzy string matching for Python";
+    homepage = "https://github.com/seatgeek/thefuzz";
+    license = licenses.gpl2;
```

No statement that later version of the license are supported.

```suggestion
    license = licenses.gpl2Only;
```